taxi/models.py

```
@@ -17,6 +18,25 @@ def __str__(self):
 class Driver(AbstractUser):
     license_number = models.CharField(max_length=255, unique=True)
 
+    def clean(self):
+        if len(self.license_number) != 8:
+            raise ValidationError(
+                "Length of License number should "
+                "be consist only of 8 characters"
+            )
+        elif not (
+                self.license_number[0:3].isalpha()
+                and self.license_number[0:3].isupper()
+        ):
+            raise ValidationError(
+                "First 3 characters should be uppercase letters"
+            )
+        elif not self.license_number[3:].isdigit():
+            raise ValidationError(
+                "Last 5 characters should be are digits"
+            )
+        return self.license_number
```

The `clean` method should not return anything. It is used to validate the model fields and should raise a `ValidationError` if any of the fields are invalid. Remove `return self.license_number`.